operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System_Using_SAT.md

```diff
@@ -0,0 +1,337 @@
+   addresses, MAC addresses, and component xnames for the nodes on the blade. If the slot was previously populated, the file passed into the
+   `--src-mapping` argument should be the mapping file saved during the
+   [Removing a Liquid-cooled Blade from a System Using SAT](Removing_a_Liquid-cooled_blade_from_a_System_Using_SAT.md) procedure, and the
+   `--dst-mapping` argument should be a mappings file of the MAC addresses, IP addresses, and component xnames from the new blade. If the new
+   blade was removed from another system, the mappings file was saved while performing the
+   [Removing a Liquid-cooled Blade from a System Using SAT](Removing_a_Liquid-cooled_blade_from_a_System_Using_SAT.md) procedure on the other system.
+
+   ```bash
+   ncn# sat swap blade --src-mapping <SRC_MAPPING> --dst-mapping <DST_MAPPING> -a enable <SLOT_XNAME>
+   ```
+
+   If the slot was not previously populated, the `--src-mapping` and `--dst-mapping` arguments should be omitted.
+
+   ```bash
+   ncn# sat swap blade -a enable <SLOT_XNAME>
+   ```
+
+## Power on and boot the nodes
+
+1. Determine which Boot Orchestration Service \(BOS\) templates to use to shut down nodes on the target blade.
+
+   There will be separate session templates for UANs and computes nodes.
+
+   1. List all the session templates.
+
+      If it is unclear which session template is in use, proceed to the next substep.
+
+      ```bash
+      ncn# cray bos sessiontemplate list
+      ```
+
+   1. Find the node xnames with `sat status`. In this example, the target blade is in slot `x9000c3s0`.
+
+      ```bash
+      ncn# sat status --filter 'xname=x9000c3s0*'
+      ```
+
+      Example output:
+
+      ```text
+      +---------------+------+----------+-------+------+---------+------+-------+-------------+----------+
+      | xname         | Type | NID      | State | Flag | Enabled | Arch | Class | Role        | Net      |
+      +---------------+------+----------+-------+------+---------+------+-------+-------------+----------+
+      | x9000c3s0b1n0 | Node | 1        | Off   | OK   | True    | X86  | River | Compute     | Sling    |
+      | x9000c3s0b2n0 | Node | 2        | Off   | OK   | True    | X86  | River | Compute     | Sling    |
+      | x9000c3s0b3n0 | Node | 3        | Off   | OK   | True    | X86  | River | Compute     | Sling    |
+      | x9000c3s0b4n0 | Node | 4        | Off   | OK   | True    | X86  | River | Compute     | Sling    |
+      +---------------+------+----------+-------+------+---------+------+-------+-------------+----------+
+      ```
+
+   1. Find the `bos_session` value for each node via the Configuration Framework Service (CFS).
+
+      ```bash
+      ncn# cray cfs components describe x9000c3s0b1n0 | grep bos_session
+      ```
+
+      Example output:
+
+      ```toml
+      bos_session = "e98cdc5d-3f2d-4fc8-a6e4-1d301d37f52f"
+      ```
+
+   1. Find the required `templateUuid` value with BOS.
```

Comment:
Does a `templateUuid` always exist in bos sessions? I only see a `templateName` on the system I'm looking at.